### PR TITLE
Closes #327 Repeat fix

### LINKED ISSUE #327: Repeat track does not repeat when clicking towards end of song
### Bug/issue:
- [x] Start playing the song
- [x] Activate the repeat track function
- [x] Click towards very end of the song
- [x] Song will not repeat

### PR DESCRIPTION
**Pull request by k----n**
### Motivation and Context
- Fixes #327 repeat issue

### Types of changes
What types of changes does your code introduce? Check all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
- [x] Accounts for the edge case where time to resume is 0, repeat is activated, and the video is still the current video being played

### Final checklist:
Go over all the following points and check all the boxes that apply  
If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](.github/CONTRIBUTING.md) guidelines.
- [x] All tests passed.

### CODE REVIEW

**Comment by k----n** [comment]
Note that this change, does not fix the autoplay issue that is resolved in #326

File: js/everything.js [hunk 1]
```diff
@@ -137,6 +137,7 @@ var ZenPlayer = {
                 that.setupTitle();
                 that.setupVideoDescription(videoID);
                 that.setupPlyrToggle();
+                that.show();
```
Comment (shakeelmohamed) [inline js/everything.js]: let's revert this change since our autoplay fix will eventually take care of it

Comment (k----n) [reply js/everything.js]: Done!

**Comment by shakeelmohamed** [comment]
Will review this when I get a chance, thanks for you effort @k----n
